lib/gelf/logger.rb

```diff
@@ -5,7 +5,7 @@ module LoggerCompatibility
     attr_accessor :formatter
 
     # Use it like Logger#add... or better not to use at all.
-    def add(level, message = nil, progname = nil)
+    def add(level, message = nil, progname = nil, &block)
```

Comment:
Okay. I did what you suggest at https://github.com/Graylog2/gelf-rb/pull/50/commits/913baae452c15dd45c2b5bdaf3a9bd38210e0cdb.

Actually I think adding back this argument wasn't a good idea from me. It have an impact on performance as having `&block` arg leads to instantiating a Proc object.

What do you think about removing it back (like it was at first place)? I'm not aware about any impact on Ruby 1.9.